src/gui/settingsDialog/uploader/ImgurUploaderSettings.cpp

```diff
@@ -50,13 +54,17 @@ ImgurUploaderSettings::~ImgurUploaderSettings()
 	delete mDirectLinkToImageCheckbox;
 	delete mAlwaysCopyToClipboardCheckBox;
 	delete mOpenLinkInBrowserCheckbox;
+	delete mUploadTitleEdit;
+	delete mUploadDescriptionEdit;
```

Comment:
I think this is not required. When parent was passed to the object then the parent will take care of deleting the object. The others can be probably deleted too.

Reply:
They all have the parent set to `ImgurUploaderSettings` and will be removed; thought this is some convention (deleting in dtor).
Do I remove the other calls to `delete` as well ? Thanks

Reply:
That's a relic from a time where I didn't know it better. You can remove all that have parent set.